src/lib/forms.js

```diff
@@ -12,6 +12,19 @@ const makeDropdown = (htmlFor, labelText, value, onChange, options, oValue, dVal
     </select>
   </label>
 );
+
+const makeYouthDropdown = (htmlFor, labelText, value, onChange, options, oValue, dValue) => (
+  <label htmlFor={htmlFor} style={{ paddingTop: '12px' }} id={htmlFor}>
+    {labelText}
+    <br />
+    <select id={htmlFor} value={value} onChange={(event) => onChange(event, 'youthPicsId')}>
```

Comment:
@Unesdala so where it has 'youthPicsId' on line 20, you would pass in the htmlFor parameter, and set that to be youthPicsId, if you can't figure it out, i'll finish it up in the morning though no problem